gsa/src/web/graphql/__mocks__/alerts.js

```diff
@@ -223,3 +226,50 @@ export const createCloneAlertQueryMock = (alertId = '1', newAlertId = '2') =>
     },
     {id: alertId},
   );
+
+const exportAlertsByIdsResult = {
+  exportAlertsByIds: {
+    exportedEntities: '<get_alerts_response status="200" status_text="OK" />',
```

Comment:
What is that? Why it is an XML string of a GMP response?

Reply:
exportedEntities is always a raw xml string of the GMP response that the frontend transforms into an xml file for download so I just mocked the first line here.